src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -9,32 +9,50 @@ import { useIntl } from "react-intl";
 
 export type Props = {
   email?: string;
+  appTheme?: string;
   lang?: string;
   hasPassword: boolean;
   updatePassword?: (password: string, passwordConfirmation: string) => void;
   updateLanguage?: (lang: string) => void;
+  updateTheme?: (lang: string) => void;
```

Comment:
Not sure updating theme with language is good :P 
```suggestion
  updateTheme?: (theme: string) => void;
```